core/web/presenters/chain.go

```diff
@@ -1,27 +1,14 @@
 package presenters
 
 import (
-	"time"
-
-	"github.com/manyminds/api2go/jsonapi"
-
 	"github.com/smartcontractkit/chainlink/core/chains"
 )
 
-type ChainResource[C chains.Config] interface {
-	IsEnabled() bool
-	GetConfig() C
-	jsonapi.EntityNamer
-}
-
-// chainResource is a generic chain resource for embedding in a ChainResource implementation.
-type chainResource[C chains.Config] struct {
+type ChainResource[C chains.Config] struct {
 	JAID
-	Enabled   bool      `json:"enabled"`
-	Config    C         `json:"config"`
-	CreatedAt time.Time `json:"createdAt"`
-	UpdatedAt time.Time `json:"updatedAt"`
+	Enabled bool `json:"enabled"`
+	Config  C    `json:"config"`
```

Comment:
I think we can do more here if/when we convert this field to be a plain string of serialized TOML.